ethcore/sync/src/chain/sync_packet.rs

```diff
@@ -22,96 +22,44 @@
 //! to convert to/from the packet id values transmitted over the
 //! wire.
 
-use api::{ETH_PROTOCOL, WARP_SYNC_PROTOCOL_ID};
-use network::{PacketId, ProtocolId};
+use syncpacketid_derive::SyncPackets;
```

Comment:
Do we really need it to be a separate crate if it's specific to SyncPackets?

Reply:
As I understand from [the documentation](https://doc.rust-lang.org/reference/procedural-macros.html#derive-mode-macros):

> Procedural macros must be defined in a crate with the crate type of proc-macro.

Besides (taken from the compiler):

> error: `proc-macro` crate types cannot export any items other than functions tagged with `#[proc_macro_derive]`

I placed it alongside `rlp-derive`, which seemed to be the right place.

Reply:
Right, sorry, I wasn't aware

Reply:
Maybe just rename it to `syncpacket` then

Reply:
No worries, I was also surprised that they had to be in their own crate.

I renamed it.